source/install/cloud-changelog.md

```diff
@@ -4,13 +4,45 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-05-25](#release-2022-05-25)
 - [Release 2022-05-12](#release-2022-05-12)
 - [Release 2022-04-28](#release-2022-04-28)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 
+## Release 2022-05-25
+
+### Highlights
+ - To make markdown features more accessible to less technical users, an Advanced Text editor was added with new shortcuts for preview, toggle formatter, opening the emoji picker, strike text, heading, ordered list, and unordered list.
```

Comment:
Proposal: Remove "to less technical users" since the new Advanced Text Editor has something for everyone regardless of technical skill/experience.